pkg/kubelet/cm/cpumanager/policy_static_test.go

```diff
@@ -33,20 +33,21 @@ import (
 )
 
 type staticPolicyTest struct {
-	description     string
-	topo            *topology.CPUTopology
-	numReservedCPUs int
-	reservedCPUs    *cpuset.CPUSet
-	podUID          string
-	options         map[string]string
-	containerName   string
-	stAssignments   state.ContainerCPUAssignments
-	stDefaultCPUSet cpuset.CPUSet
-	pod             *v1.Pod
-	topologyHint    *topologymanager.TopologyHint
-	expErr          error
-	expCPUAlloc     bool
-	expCSet         cpuset.CPUSet
+	description      string
+	topo             *topology.CPUTopology
+	numReservedCPUs  int
+	reservedCPUs     *cpuset.CPUSet
+	cpuPolicyOptions map[string]string
```

Comment:
the existing `options` field is meant to be used to hold cpu policy options, no need to duplicate it